dsc/decisionMaker.md

```diff
@@ -0,0 +1,36 @@
+# Desired State Configuration Overview for Decision Makers #
```

Comment:
I have yet to write up the style guide for our Markdown guidelines, but I believe we've coalesced around not having the trailing pound signs. GitHub renders fine with just the leading pound signs, and I don't see any real reason to include the trailing ones.